portfolio/src/main/java/com/google/sps/Comment.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps;
+
+public final class Comment {
+
+  private final long id;
+  private final String name;
+  private final String text;
+  private final long timestamp;
+
+  public Comment(long id, String name, String text, long timestamp) {
```

Comment:
I'm not asking for any changes now, but here is something to start thinking about.

This constructor has four parameters: two longs and two strings. The duplicate types means it's possible to pass values in the wrong order, for example I could accidentally write

```
new Comment(id, text, name, timestamp)
```

Any thoughts about how to make this harder to misuse?